src/ansys/mechanical/core/embedding/viz/pyvista_plotter.py

```diff
@@ -68,9 +69,3 @@ def to_pyvista_plotter(app: "ansys.mechanical.core.embedding.App"):
         color = pv.Color(bgr_to_rgb_tuple(body.Color))
         plotter.add_mesh(polydata, color=color, smooth_shading=True)
```

Comment:
You probably want to use `plot` method here, extra kwargs are passed downstream to the PyVista backend.
```suggestion
        plotter.plot(polydata, color=color, smooth_shading=True)
```